src/main/java/de/tum/cit/aet/artemis/iris/dao/IrisChatSessionDAO.java

```diff
@@ -1,10 +1,13 @@
 package de.tum.cit.aet.artemis.iris.dao;
 
+import jakarta.annotation.Nullable;
+import jakarta.validation.constraints.NotNull;
+
 import com.fasterxml.jackson.annotation.JsonInclude;
 
 import de.tum.cit.aet.artemis.iris.domain.session.IrisChatSession;
 
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
-public record IrisChatSessionDAO(IrisChatSession session, long entityId) {
+public record IrisChatSessionDAO(@NotNull IrisChatSession session, @NotNull long entityId, @Nullable String entityName) {
```

Comment:
Native types cannot be null, or am I missing something?

```suggestion
public record IrisChatSessionDAO(@NotNull IrisChatSession session, long entityId, @Nullable String entityName) {
```